src/ufoLib2/objects/misc.py

```diff
@@ -135,10 +136,34 @@ class DataStore(MutableMapping):
 
     _data: Dict[str, Union[bytes, Placeholder]] = attr.ib(factory=dict)
 
+    _lazy: Optional[bool] = attr.ib(default=None, kw_only=True, cmp=False)
```

Comment:
shouldn't `_lazy` also be `init=False`?

Reply:
and perhaps it should default to False, I don't think `default=None` makes any difference in the case of the DataStore.

Reply:
A data store is lazy by default because `Font` is lazy by default, right? I just mirror the setup from `Font`.